manifests/install.pp

```diff
@@ -213,6 +213,9 @@
       if $::python::version =~ /^3/ {
         $pip_category = undef
         $pip_package = 'python3-pip'
+      } elsif ($::osfamily == 'RedHat') and (versioncmp($::operatingsystemmajrelease, '7') >= 0) and ($python::use_epel == true) {
```

Comment:
I disabled use_epel since I have manage it elsewhere, but still would like this case to apply, which is why I like #349 better.